src/reverselock.h

```diff
@@ -13,7 +13,7 @@ class reverse_lock
 {
 public:
 
-    explicit reverse_lock(Lock& lock) : lock(lock) {
+    explicit reverse_lock(Lock& _lock) : lock(_lock) {
```

Comment:
I'll take a look. Aand maybe release the script, though it's a Chthonic monstriosity with tentacles spreading to other unpublished tools, so I have to figure out what to collect along the way.